src/stave.ts

```diff
@@ -80,14 +80,12 @@ export class Stave extends Element {
   // This is the sum of the padding that normally goes on left + right of a stave during
   // drawing. Used to size staves correctly with content width.
   static get defaultPadding(): number {
-    const musicFont = Tables.currentMusicFont();
-    return musicFont.lookupMetric('stave.padding') + musicFont.lookupMetric('stave.endPaddingMax');
+    return Tables.lookupMetric('Stave.padding') + Tables.lookupMetric('Stave.endPaddingMax');
```

Comment:
Once migration is done, we should figure out where the `lookupMetric()` function belongs. I guess it's OK to make the `Tables` class officially public, as long as we clean it up and make the API clear & easy to use.